### ISSUE TITLE
output files 

### ISSUE THREAD

**Issue by 0Mrbit1** [0Mrbit1/Minishell#10]
Output files can be many , need to be handled by the parser

**Comment by anas-jmp** [comment]
yep  ,  the solution is to change it to an array for multiple Output  files
